backend/distributor/migrations/0035_import_hospitals_addresses.py

```diff
@@ -0,0 +1,38 @@
+import csv
+
+from django.db import migrations, transaction
+
+from distributor.models import Hospital, Statistic, StatisticCategory
```

Comment:
We can't import the models directly as it may be a newer version than this migration expects. We use the historical version.

Reply:
it gonna be like this `Hospital = apps.get_model('distributor', 'Hospital')`